src/components/Modals/Settings/SettingsModal.tsx

```diff
@@ -110,6 +149,71 @@ export const SettingsModal: React.FunctionComponent = () => {
         </div>
       ),
     },
+    {
+      name: 'Governance',
+      tab: <span>Governance</span>,
+      content: (
+        <>
+          <div>
+            <div className="flex flex-col mt-4 text-sm">
+              <div className="flex flex-row content-center justify-between">
+                <label
+                  htmlFor="settings-governance-show-warnings"
+                  className="flex justify-right items-center w-1/2 content-center font-medium text-gray-700"
+                >
+                  Show&nbsp;<strong>warning</strong>&nbsp;governance issues:
+                </label>
+                <Switch
+                  toggle={governanceWarnings}
+                  onChange={setGovernanceWarnings}
+                />
+              </div>
+              <div className='text-gray-400 text-xs'>
+                Show warning governance issues in the editor&apos;s&nbsp;<strong>Diagnostics</strong>&nbsp;tab.
+              </div>
+            </div>
+          </div>
```

Comment:
I think you can already convert this into a component. It's repeated 3 times.

Reply:
Done :)